flake8_django/checkers/model_content_order.py

```diff
@@ -8,7 +8,7 @@
 class DJ12(Issue):
     code = 'DJ12'
     description = (
-        "Order of Model's inner classes, methods, and fields does not follow the"
+        "The order of the model's inner classes, methods, and fields do not follow the "
```

Comment:
Thanks for making this change, I like the new wording :). Slight gramma fix:

```suggestion
        "The order of the model's inner classes, methods, and fields does not follow the "
```

Reply:
I *think* do is correct when describing multiple objects ([ref](https://english.stackexchange.com/questions/30575/do-vs-does-with-multiple-singular-subjects)) but could be wrong :) I'll commit the suggestion for now

Reply:
In this case, the subject is "The Order of ..." which is singular, that's why I think it should be `does` (I could be wrong).

Reply:
Not a big deal either way 😄